e2e-tests/cypress/integration/appeal-submission-appeal-site-address.feature

```diff
@@ -32,35 +35,21 @@ Feature: Appellant provides the Appeal Site Address
     Examples:
       | Address Line 1 | Address Line 2 | Town or City | County       | Postcode   | reason                       |
       | ""             | ""             | ""           | "South Glos" | "W1A 1HQ"  | "Address Line 1 is required" |
-      | "aaa"          | ""             | ""           | ""           | "EC1A 1BB" | "County is required"         |
-      | "aaa"          | ""             | ""           | "South Glos" | ""         | "Postcode is required"       |
+      | "aaa"          | ""             | ""           | ""           | ""         | "Postcode is required"       |
 
-  @as-1680
+  @as-1680 @as-2483
   Scenario: Prospective appellant fails to provide any address information
     Given the user is prompted for the site address
     When the user provides their appeal site address as "" and "" and "" and "" and ""
     Then the user is informed that "Address Line 1 is required"
-    And the user is informed that "County is required"
     And the user is informed that "Postcode is required"
 
-  Scenario: Prospective appellant fails to provide first address line and county
-    Given the user is prompted for the site address
-    When the user provides their appeal site address as "" and "" and "" and "" and "BS8 1TG"
-    Then the user is informed that "Address Line 1 is required"
-    And the user is informed that "County is required"
-
   Scenario: Prospective appellant fails to provide first address line and Postcode
     Given the user is prompted for the site address
     When the user provides their appeal site address as "" and "" and "" and "South Glos" and ""
     Then the user is informed that "Address Line 1 is required"
     And the user is informed that "Postcode is required"
 
-  Scenario: Prospective appellant fails to provide County and Postcode
-    Given the user is prompted for the site address
-    When the user provides their appeal site address as "1 Taylor Road" and "" and "" and "" and ""
-    Then the user is informed that "County is required"
-    And the user is informed that "Postcode is required"
-
```

Comment:
this and the other removed scenario (46-51) are covered by `Prospective Appellant submits an appeal site address without mandatory information` from my pov